<?xml version="1.0" encoding="UTF-8"?>
<launch>
    <include file="$(find fabrics_bridge)/launch/fabrics_node.launch" />

    <node name="interactive_marker" pkg="fabrics_bridge" type="interactive_marker.py">
        <param name="link_name" value="panda_link0" />
        <remap to="/fabrics/planning_goal" from="equilibrium_pose" />
    </node>

</launch>
